src/settings.cpp

```diff
@@ -1527,8 +1527,7 @@ QString ShotcutSettings::whisperModel()
 {
     QDir dataPath = QmlApplication::dataDir();
     dataPath.cd("shotcut/whisper_models");
-    return settings.value("subtitles/whisperModel", dataPath.absoluteFilePath("ggml-base-q5_1.bin"))
-        .toString();
+    return settings.value("subtitles/whisperModel", "").toString();
```

Comment:
Is it your intention to not include a default model with the downloaded package any longer? It is still bundling but no longer recognized from the install. Existing users are in for a shock when they see the red text: Select a model or browse to choose one--especially if they are updating an existing install. That's somewhat fine, but I need to change the build scripts. Do you think we should make the old, bundled model bold in the list to make it easier to get back to what you had before? Otherwise, I worry people will not know what to do and simply choose the first.

Reply:
> Is it your intention to not include a default model with the downloaded package any longer?

That is my intention. But I had not changed the build script yet to give more time for testing.

> Otherwise, I worry people will not know what to do and simply choose the first.

Well, that is a very good comment that I did not think of. I think it would be good to remove it from the build script.

To help the user, I can change the text in the list to say "Base Multilingual Quantized 5_1 (Recommended)"

Reply:
I'm not sure that this will send the message we are looking for:
![image](https://github.com/user-attachments/assets/72f921b3-311d-4fc2-82cf-d29c8ce3f043)

Another idea I have: When we open the dialog, we could detect if they do not have any models downloaded. In that case, we could pop up a prompt that says:

"You have not downloaded any models. Click a model to download one. Would you like to download a common model to get started?"

If they click "yes", we would start the download for them.

Reply:
This is done. I also added a way to delete models.